website/notifications/views.py

```diff
@@ -0,0 +1,51 @@
+from framework.auth.decorators import must_be_logged_in
+from model import Subscription
+from flask import request
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from modularodm.storage.mongostorage import KeyExistsException
+from website import settings
+
+@must_be_logged_in
+def subscribe(auth):
+    user = auth.user
+    subscription = request.json
+    event = subscription.get('event')
```

Comment:
What if `event` and `notification_type` are not provided by the client? Should we return an error response?